### PR TITLE
:sparkles: feat(annotation): add DomainService annotation

### PR DESCRIPTION
**Pull request by Yugo-Cartron**
# Pull request for Candy-doc

## Your checklist for this pull request

🚨 Please review the [guidelines for contributing](../../CONTRIBUTING.md) to
this repository.

- [x] Make sure you are requesting to **pull a feature/bugfix branch** (right
  side). Don't request your master!
- [x] Make sure you are making a pull request against the **main branch** (left
  side).
- [x] Check the commit's or even all commits' message styles matches our
  requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit
  test.

## Description

Add @DomainService annotation for the plugin

❤️ Thank you!

### CODE REVIEW

File: src/main/java/io/candydoc/ddd/tactical_pattern/DomainService.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+
+@SuperBuilder
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class DomainService extends TacticalModel {
+
+  private static final Set<DDDPattern> ALLOWED_DEPENDENCIES =
```
[the remark this answers is not in the document]
Comment (Yugo-Cartron) [reply src/main/java/io/candydoc/ddd/tactical_pattern/DomainService.java:14]: We'll add aggregate and commands as dependencies. Repository annotation belongs to another branch.

[the remark this answers is not in the document]
Comment (Yugo-Cartron) [reply src/main/java/io/candydoc/ddd/tactical_pattern/DomainService.java:14]: As you noticed, distinguish the kind of dependency is becoming crucial. We're actually working on it.

**Review by ecattez — DISMISSED** [review]

**Review by ctacquet — APPROVED** [review]

**Review by dylanmsql — APPROVED** [review]
Bien ouej bg

**Pull request merged**